apisix/admin/init.lua

```diff
@@ -253,8 +254,9 @@ end
 
 local function get_plugins_list()
     set_ctx_and_check_token()
-
-    local plugins = resources.plugins.get_plugins_list()
+    local args = get_uri_args()
+    local subsystem = args["subsystem"]
+    local plugins = resources.plugins.get_plugins_list(subsystem)
```

Comment:
Should we check the value of subsystem?

Reply:
htt&stream could not be accepted, it's not useless

Reply:
done